src/components/common/Extensions/MainNavigation/partials/SubMenu/SubMenu.tsx

```diff
@@ -21,12 +21,7 @@ const SubMenu: FC<SubMenuProps> = ({ items }) => {
 
   return (
     <Wrapper hasShadow={!isMobile}>
-      <ul
-        className={clsx(
-          styles.subMenuList,
-          'sm:grid-cols-[repeat(2,1fr)] md:grid-cols-[repeat(3,1fr)]',
-        )}
-      >
+      <ul className={clsx(styles.subMenuList, 'grid-cols-[repeat(3,1fr)]')}>
```

Comment:
`grid-cols-[repeat(3,1fr)]` => `grid-cols-3`

Reply:
It's not the same and it doesn't work with `grid-cols-3`